src/transaction.ts

```diff
@@ -59,14 +59,15 @@ export interface ExecuteSqlRequest extends Statement, RequestOptions {
   partitionToken?: Uint8Array|string;
   seqno?: number;
 }
-
 export interface KeyRange {
   startClosed?: Value[];
   startOpen?: Value[];
   endClosed?: Value[];
   endOpen?: Value[];
 }
-
+export interface ReadCallback {
+  (err: null|ServiceError, rows: Rows): void;
```

Comment:
I assume rows can be undefined or null, so we might want to use the `RequestCallback` generic here.